[FAQ] fix ledger privacy leak details (#566)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

You can't do that directly, you have to send the bitcoins (in small portions > 0.1 BTC if needed) to a `hot` Wasabi Wallet, do the CoinJoin and then send them back to a new address on the Hardware wallet for cold-storage.
:::

:::details

> ### Does Ledger Live send my public keys to a third party server?

Reviewer: ```suggestion
### Does Ledger Live send my public keys (addresses) to a third party server?
```

> ### Does Ledger Live send my public keys (addresses) to a third party server?

Author: ```suggestion
### Does Ledger Live send my public keys and addresses to a third party server?
```

Yes.

> When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused future public keys to the Ledger company server.

Reviewer: ```suggestion
When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused public keys to the Ledger company server.
```

> When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused public keys to the Ledger company server.

Reviewer 2: 'you send all of your used, and 20 unused public keys to the Ledger company server' 
Missing any source or proof of this claim. Change without any explanation doesn't sound so reliably. It is exact so I believe there is some test of Ledger Live behind the claim. Maybe add link to some test or source where reader can verify this.

Reviewer 2: Address gap limit set to 20 unused addresses is described in BIP 44. 

https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#address-gap-limit

> When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused public keys to the Ledger company server.

Reviewer 2: ```suggestion
When using the Ledger Live software wallet to manage your coins, you send [all of your used, and 20 unused public keys ](https://support.ledger.com/hc/en-us/articles/360010892360-Address-gap-limit)to the Ledger company server. 
```

> When using the Ledger Live software wallet to manage your coins, you send all of your used, and [20 unused public keys](https://support.ledger.com/hc/en-us/articles/360010892360-Address-gap-limit) to the Ledger company server.

Reviewer: ```suggestion
When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused public keys to the Ledger company server. [Source](https://support.ledger.com/hc/en-us/articles/360010892360)
```

With this information they know exactly how many bitcoins you have and in what transactions you spend them.

> Your extended public key, however, is not shared with this server, but rather stored encrypted on your local machine.

Reviewer: ```suggestion
Your extended public key, however, is not shared with this server, but rather stored encrypted on your local machine. [Source](https://support.ledger.com/hc/en-us/articles/360011069619)
```

> To avoid this privacy leak, you can use a Ledger hardware in combination with Wasabi as software interface, and because Wasabi does not leak your public keys, your transaction history is not shared with anyone.

Reviewer: ```suggestion
To avoid this privacy leak, you can use a Ledger hardware in combination with Wasabi as a software interface, and because Wasabi does not leak your public keys, your transaction history is not shared with anyone.
```
